src/gemm_v2.cpp

```diff
@@ -36,10 +36,14 @@
 #endif
 
 #if MIOPEN_USE_ROCBLAS
+#if defined(_WIN32) && (HIP_PACKAGE_VERSION_FLAT < 5007000000ULL)
```

Comment:
@apwojcik  IIUC this means the `ROCBLA_BETA_FATURES_API` would always be enabled on Linux, regardless of the HIP version, which is not what we want ?